CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.44.0-1
+* Cache Agent version 1.44.0
```

Comment:
you need to add some details about teresa's logging driver support feature since it was a change in init as well

Reply:
for example see the entry for 1.42.0-1.

Reply:
Done